website/content/docs/operations/metrics.mdx

```diff
@@ -0,0 +1,92 @@
+---
+layout: docs
+page_title: Boundary Metrics
+description: |-
+  Obtain visibility of various components of a running Boundary
+---
+
+## Boundary Metrics
+
+Boundary provides metrics through the `/metrics` path using a listener with the
+`"ops"` purpose. By default a listener with that purpose runs on port `9203`.
+See the [example configuration](#example-configuration) section for an example
+listener stanza in a `config.hcl` file.
+
+Metrics are important for understanding what is happening in a boundary
+deployment, enabling diagnoses of deployed controllers and workers. The exported
+metrics grant greater visibility into the various components of a running
+Boundary deployment through Prometheus.
+
+For more information about Prometheus, see the [Prometheus Overview
+Guide](https://prometheus.io/docs/introduction/overview). Boundary uses the
+[OpenMetric
+exposition](https://prometheus.io/docs/instrumenting/exposition_formats/#openmetrics-text-format)
+format.
+
+### Controller
+
+The following lists the current names and descriptions of metrics provided for controllers.
+
+| Name                                                          | Description                                   |
+|---------------------------------------------------------------|-----------------------------------------------|
+| `boundary_controller_api_http_request_duration_seconds`       | Historgram of latencies for HTTP requests.     |  
+| `boundary_controller_api_http_request_size_bytes`             | Histogram of request sizes for HTTP requests.  |
+| `boundary_controller_api_http_response_size_bytes`            | Histogram of response sizes for HTTP requests. |
+| `boundary_controller_cluster_grpc_request_duration_seconds`   | Histogram of latencies for requests made to the gRPC service running on the cluster listener. |
+
+### Worker
+
+The following lists the current names and descriptions of metrics provided for
+workers.
+
+| Name                                                          | Description                                   |
+|---------------------------------------------------------------|-----------------------------------------------|
+| `boundary_worker_proxy_http_write_header_duration_seconds`    | Histogram of time elapsed after the TLS connection is established to when the first http header is written back from the server. |
+| `boundary_worker_proxy_websocket_active_connections`          | A gauge of the current count of open proxy connections on the worker. |
+| `boundary_worker_proxy_websocket_received_bytes_total`        | Count of received bytes sent over all proxy connections handled by the worker. |
+| `boundary_worker_proxy_websocket_sent_bytes_total`            | Count of sent bytes  sent over all proxy connections handled by the worker. |
+
+
```

Comment:
```
### Other

The following current miscellaneous metrics provided for Boundary.

| Name                                                          | Description                                   |
|---------------------------------------------------------------|-----------------------------------------------|
| `boundary_build_info`    | A gauge set to 1 with label values detailing the current Go version, git revision, and Boundary version. |
```

Do we want something like this detailing the current build info metric? 

Maybe we take out "with label values" since labels are introduced right after.

Does that mean we will need a label table with the 3 build info labels in the following section? I think that would be extraneous since this description would already get the point across.

@talanknight what do you think?

Reply:
I think we should be explicit and include both the "Other" block shown above as well as a label table for this metric. And I think we should then include examples of how this can be used along with the other metrics in the TBD tutorial.

Reply:
Sounds good. I wrote a template for the labels block in the [doc draft](https://docs.google.com/document/d/11cQycnZYk0B2c2N52ae1CfRE06Chfwvuo6wvSqIBwck/edit).
For the examples, I think Robin is covering that in his learn guide tutorial

Reply:
@A-440Hz, how does the new Other section look for `boundary_build_info`?

Reply:
Looks good, thanks. I think I made a typo though. Could you correct it to "The following lists the current miscellaneous metrics provided for Boundary"?